packages/nodes-base/nodes/Onfleet/descriptions/TaskDescription.ts

```diff
@@ -0,0 +1,431 @@
+import {
+	INodeProperties
+} from 'n8n-workflow';
+
+export const taskOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'tasks',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Add',
```

Comment:
This should be `create` as `add` is a bit confusing

Reply:
Name changed